mobile/native/src/common/mod.rs

```diff
@@ -0,0 +1,14 @@
+use flutter_rust_bridge::frb;
+
+#[frb]
```

Comment:
Given this has `frb` on it, should this model only live in `api_model`?

Reply:
I did a cleanup round and removed the `common` module, not worth having, better encapsulate *all* the types from the get-go. If we feel the boileplate is not worth it we can change that again easily.

Note that `api_model` is nicely encapsulated, besides the notifications - the BL imports types from the API layer there which is not optimal (higher layer should know lower layer, not the other way round...). But I would merge this for now and iterate on it :)